kite-morphlines/kite-morphlines-maxmind/src/test/java/org/kitesdk/morphline/maxmind/MaxmindMorphlineTest.java

```diff
@@ -15,75 +15,150 @@
  */
 package org.kitesdk.morphline.maxmind;
 
+import org.junit.After;
 import org.junit.Ignore;
 import org.junit.Test;
 
 import org.kitesdk.morphline.api.AbstractMorphlineTest;
+import org.kitesdk.morphline.api.Command;
+import org.kitesdk.morphline.api.MorphlineCompilationException;
+import org.kitesdk.morphline.api.MorphlineContext;
 import org.kitesdk.morphline.api.Record;
 import org.kitesdk.morphline.base.Fields;
 import org.kitesdk.morphline.base.Notifications;
+import org.kitesdk.morphline.stdlib.PipeBuilder;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
 
 public class MaxmindMorphlineTest extends AbstractMorphlineTest {
 
+  private Command buildCommand(final String inputField, String db) {
+    final Config config = ConfigFactory.parseString(
+        String.format("inputField=%s\ndatabase=target/test-classes/%s\n", inputField, db)
+    );
+    return new GeoIPBuilder().build(config, new RootCommand(), collector, new MorphlineContext.Builder().build());
+  }
+
+  @After
+  public void tearDown() {
+    if (morphline != null) {
+      Notifications.notifyShutdown(morphline);
+      Notifications.notifyShutdown(morphline);
+      morphline = null;
+    }
+  }
+
   @Test
   public void testIPv4() throws Exception {
-    morphline = createMorphline("test-morphlines/geoIP");    
-    
-    Record record = new Record();
-    String ip = "128.101.101.101";
+    morphline = buildCommand("ip", "GeoLite2-City.mmdb");
+    final Record record = new Record();
+    final String ip = "128.101.101.101";
     record.put("ip", ip);
-    Record expected = new Record();
-    expected.put("ip", ip);
-    expected.put("/country/iso_code", "US");
```

Comment:
Can we keep the original unit test case and just add new test cases for the new additional behaviour?